pkg/k8shandler/cluster.go

```diff
@@ -46,8 +46,7 @@ func CreateOrUpdateElasticsearchCluster(dpl *v1alpha1.Elasticsearch, configMapNa
 	if err != nil {
 		return err
 	}
-	// TODO: get rid of this message as part of LOG-206
-	logrus.Infof("cluster %s required action is: %v", dpl.Name, action)
+	logrus.Debugf("cluster %s required action is: %v", dpl.Name, action)
```

Comment:
Is there a reason to even print this out? Can't we discern what the action is based on the elasticsearch object's status field?